backend/plugins/frontline_api/src/modules/integrations/call/graphql/resolvers/queries.ts

```
@@ -0,0 +1,238 @@
+            extension: queue,
+          },
+        },
+        integrationId: integrationId,
+        retryCount: 3,
+        isConvertToJson: true,
+        isAddExtention: false,
+      },
+      user,
+    )) as any;
+
+    if (queueData && queueData.response) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)